src/test/java/org/opensearch/security/auditlog/AbstractAuditlogiUnitTest.java

```diff
@@ -83,34 +83,27 @@ protected void setupStarfleetIndex() {
         rh.keystore = keystore;
     }
 
-    protected boolean validateMsgs(final Collection<AuditMessage> msgs) {
-        boolean valid = true;
+    protected void validateMsgs(final Collection<AuditMessage> msgs) throws Exception {
         for (AuditMessage msg : msgs) {
-            valid = validateMsg(msg) && valid;
+            validateMsg(msg);
         }
-        return valid;
-    }
 
-    protected boolean validateMsg(final AuditMessage msg) {
-        return validateJson(msg.toJson()) && validateJson(msg.toPrettyString());
     }
 
-    protected boolean validateJson(final String json) {
+    protected void validateMsg(final AuditMessage msg) throws Exception {
+        validateJson(msg.toJson());
+        validateJson(msg.toPrettyString());
```

Comment:
why do we need to validate twice?

Reply:
I'm not sure what toPrettyString does so I'm assuming this tests if both versions of the msg are valid. This was in the previous implementation, do we need this? @DarshitChanpura

Reply:
@DarshitChanpura Following up on this, please also allow the tests to run so I can see what errors pop up. Thanks!